ocs_ci/deployment/acm.py

```diff
@@ -157,6 +157,27 @@ def download_binary(self):
                 os.path.expanduser("~/.local/bin/subctl"),
                 os.path.join(config.RUN["bin_dir"], "subctl"),
             )
+        elif self.source == "downstream":
+            self.download_downstream_binary
+
+    def download_downstream_binary(self):
+        subctl_ver = config.ENV_DATA["subctl_version"]
+        cmd = (
+            f"oc image {constants.SUBCTL_DOWNSTREAM_URL}{subctl_ver} "
+            f'--path="/dist/{subctl_ver}*-linux-amd64.tar.xz":/tmp --confirm'
```

Comment:
I think we should get this amd64 dynamically based on currently running platform. As we can run ocs-ci on Apple Silicone chips like M1 - M2 ... we should consider to support also this arm64 binairies.

Reply:
You can check this: https://stackoverflow.com/questions/65970469/what-does-platform-system-and-platform-architecture-return-on-apple-m1-silic

Reply:
for now just added the conditional statement for x86_64 processor, I think we need a list of all supported ocs-ci platform names, ofcourse we need to check platforms for which binaries are available.